src/antidote.erl

```diff
@@ -307,30 +337,14 @@ read(Key, Type) ->
 -spec clocksi_execute_tx(Clock :: snapshot_time(),
                          [client_op()],snapshot_time(),boolean()) -> {ok, {txid(), [snapshot()], snapshot_time()}} | {error, term()}.
 clocksi_execute_tx(Clock, Operations, UpdateClock, KeepAlive) ->
```

Comment:
This function will be deprecated once I apply the same changes to reads.
This is, making reads parallel, and making read_only_transactions be handled entirely by the interactive txn coord.